src/org/zaproxy/zap/extension/fieldenumeration/ZapAddOn.xml

```diff
@@ -0,0 +1,18 @@
+<zapaddon>
+	<name>fieldenumeration</name>
+	<version>1</version>
+	<status>alpha</status>
+	<description>Provides blacklist characters</description>
+	<author>ZAP Dev Team</author>
+	<url></url>
+	<changes></changes>
```

Comment:
Should include a single entry, like `Initial release. Addresses issue 1405.` or something like that.